test/dynamo/test_functions.py

```diff
@@ -217,6 +217,17 @@ def test_slice5(a):
     def test_slice6(a):
         return torch.unsqueeze(a, 0)[:, 2:]
 
+    @make_test
+    def test_range1(a):
+        return torch.tensor(range(a.size(0)))
+
+    @make_test
+    def test_range2(x, y):
+        r = x + y
+        for i in range(x.size(0) + 2):
+            r = r / y
+        return r
+
```

Comment:
There are ```test_restore_range``` and ```test_restore_range_iter``` in ```test_subgraphs.py``` used to test ```range``` works well after graph break, so only add these two to cover this bug fix.